src/util/Collection.js

```diff
@@ -60,58 +60,110 @@ class Collection extends Map {
 
   /**
    * Obtains the first item in this collection.
-   * @returns {*}
+   * @param {Integer} [count] Number of values to return. If omited, returns a single values.
```

Comment:
Param should be of type `number` instead of `Integer` as the latter won't resolve to anything.

This applies to all subsequent ones obviously.